k8s/service_controller/Makefile

```diff
@@ -14,10 +14,10 @@ binary: $(EXE)
 $(EXE): $(SRC_FILES)
 ifeq (,$(shell which go))
 	@$(MAKE) --no-print-directory binary-docker
+	docker build -t $(IMAGE):latest .
 else
 	@$(MAKE) --no-print-directory binary-local
 endif
-	docker build -t $(IMAGE):latest .
```

Comment:
we won't be able to avoid it at some point since we'll need to run kube for our tests and right now I think that means hyperkube in a container(s).  Embrace the inevitable  :-)